### PR TITLE
Prefer Ayatana AppIndicator but also support Ubuntu Appindicator

### PR DESCRIPTION
**Pull request by lukefromdc**

### CODE REVIEW

**Comment by lukefromdc** [comment]
This is the updated version of https://github.com/mate-desktop/mate-polkit/pull/41/

**Comment by lukefromdc** [comment]
Just tested successfully with Ayatana appindicator, since I cannot readly uninstall this a test against the older Ubuntu appindicator is not practical here. Previously had both installed at once

**Comment by lukefromdc** [comment]
Cleaned up a tab used instead of spaces by mistake. 
We NEED this if indicators are to be used in Debian Unstable or the coming release witthout digging up years-old packages

File: src/main.c [hunk 1]
```diff
@@ -42,6 +43,7 @@
 #define SM_DBUS_INTERFACE "org.gnome.SessionManager"
 #define SM_CLIENT_DBUS_INTERFACE "org.gnome.SessionManager.ClientPrivate"
 
+
```
Comment (raveit65) [inline src/main.c:46]: Why adding here a second blank line?

**Comment by raveit65** [comment]
It builds fine with using libappindicator on a  ayatana appIndicator free system like fedora.
```
    AppIndicator enabled:              yes
    Ayatana AppIndicator (preferred)   no
    Ubuntu AppIndicator (legacy)       yes
```

**Review by raveit65 — APPROVED** [review]
Looks good beside from the unneeded new blank line in main.c

**Comment by raveit65** [comment]
As a side note i found out that it is possible to to use both build dependencies when installed.
```
    AppIndicator enabled:              yes
    Ayatana AppIndicator (preferred)   yes
    Ubuntu AppIndicator (legacy)       yes
```
Well, i don't think that there is a distro which use both but it is  a bit weird.

File: src/main.c (hunk 1 above)
Comment (lukefromdc) [reply src/main.c]: By accident, taking it out and will force push

**Review by lukefromdc — COMMENTED** [review]
removed that line, retested and no issues as expected, force-pushed

**Pull request merged**